sdcm/cluster.py

```diff
@@ -383,6 +384,25 @@ def install_scylla_bench(self):
         """))
         self.is_scylla_bench_installed = True
 
+    @retrying(n=3)
+    def install_cassandra_harry(self):
+        if self.distro.is_rhel_like:
+            self.remoter.sudo("yum install -y git make maven")
+        else:
+            self.remoter.sudo(shell_script_cmd("""\
+                apt-get update
+                apt-get install -y git make maven
+            """))
+        self.remoter.run(shell_script_cmd("""\
+            rm -rf cassandra-harry
+            git clone -b scylla-branch https://github.com/amoskong/cassandra-harry
```

Comment:
Amos sent a PR to apache, but for now we can use his fork till they merge it.
I'll create a task reminder for that.